Makefile

```
@@ -19,11 +19,15 @@ strip: build
 clean:
 	rm pbapi pbmigrate
 
+
+DOCKER_BUILD_OPTS ?= "--build-arg quay_expiration=2d"
+DOCKER_IMAGE ?= "provisioning-backend"
```

Also, given we don't build by docker, should we name them `CONTAINER_*`
```suggestion
CONTAINER_BUILD_OPTS  ?= "--build-arg quay_expiration=2d"
CONTAINER_IMAGE_NAME ?= "provisioning-backend"
```